_pages/terminology-server.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Terminology Server
+permalink: /terminology-server/
+---
+
+TIMS hosts a FHIR server that provides a unique set of ontologies, terminologies, with unique features such as text 
+search and  SSSOM-formatted mappings.
+
+- http://fhir.terminology-services.com
+- http://20.3.198.176:8080/ (mirror)
+
+### Notable features
+[//]: # (TODO: need to clarify this list)
+- Content: CodeSystem and ConceptMap resources (see more below), and ValueSets (in development).
+- CodeSystem operations: [$subsumes](https://build.fhir.org/codesystem-operations.html#subsumes), [$lookup](https://build.fhir.org/codesystem-operations.html#lookup)
+- ConceptMap operations: [$closure](https://build.fhir.org/conceptmap-operations.html#closure) (in development)
+- ValueSet operations: [$expand](https://build.fhir.org/valueset-operations.html#expand), [$validate-code](https://build.fhir.org/valueset-operations.html#validate-code) (in development)
+
+#### Unique offerings
```

Comment:
@ShahimEssaid I could use your help verifying that what I have in this section looks good / is correct.